app/sockets.js

```diff
@@ -383,16 +407,72 @@ async function newsletterSubscribe(data, socket) {
       `> _Cause: ${email} interacted with the form_\n`
     );
 
-    return socket.send(JSON.stringify({
+    return send(socket, {
       class: "error",
       html: "Something is terribly wrong",
       message: "updated html",
       selector: "#emailMessage"
-    }));
+    });
+  }
+}
+
+export function send(transport, data) {
+  return transport.send(JSON.stringify(data));
+}
+
+async function syncWithApi(data, socket) {
+  const tokenResponse = await verifyGitHubToken(data.code);
+
+  if (tokenResponse === null) {
+    return send(socket, {
+      html: "<p><strong>There was an issue with accessing GitHub's API. Please try again later.</strong></p>",
+      message: "updated html",
+      selector: "developer-program"
+    });
+  }
+
+  try {
+    let result = await got(`https://${apiUrl}/reward/new?github_token=${tokenResponse}&reward_type=github_developer&wallet_address=${data.address}`, { json: true });
+
+    result = result.body.data;
+
+    return send(socket, {
+      html: `<p><strong>Success!</strong> Your wallet has been credited with ${result.reward_amount} LBC.</p><p>We have a great reference for the <a href="/api/sdk">LBRY SDK here</a> to help you get started.</p>`,
```

Comment:
I was never able to get this to work, but I assume it actually does. It would be good to update this to include a link to the transaction on the block explorer.

Reply:
It requires a test wallet address.

Reply:
Ah, just realized you must've been trying to do this with the production values. I'm not sure if it's active or not.

Reply:
I'm not sure how to grab the block explorer transaction since I have only been doing this in testing and I don't recall seeing an address I could use. I may forgetting this though.